dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNBusInterfaceIIDM.cpp

```diff
@@ -130,7 +131,13 @@ BusInterfaceIIDM::getComponentVarIndex(const std::string& varName) const {
 void
 BusInterfaceIIDM::exportStateVariablesUnitComponent() {
   busIIDM_.setV(getStateVarV());
```

Comment:
Why limiting this safer approach to the angle and not applying it also to the voltage (as implied by the PR's title by the way that mentions "voltages equal to 0")

Reply:
The issue was observed on the angle, so I preferred to "play safe" to limit the effects